warehouse/templates/manage/account/publishing.html

```diff
@@ -113,100 +209,17 @@ <h2>{% trans %}Add a new pending publisher{% endtrans %}</h2>
-    {{ form_errors(pending_github_publisher_form) }}
-    <div class="form-group">
-      <label for="project_name" class="form-group__label">
-        {% trans %}PyPI Project Name{% endtrans %}
-        {% if pending_github_publisher_form.project_name.flags.required %}
-        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
-        {% endif %}
-      </label>
-      {{ pending_github_publisher_form.project_name(placeholder=gettext("project name"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="project_name-errors") }}
-      <p class="form-group__help-text">
-        {% trans %}The project (on PyPI) that will be created when this publisher is used{% endtrans %}
-      </p>
-      <div id="project_name-errors">
-        {{ field_errors(pending_github_publisher_form.project_name) }}
-      </div>
-    </div>
-    <div class="form-group">
-      <label for="owner" class="form-group__label">
-        {% trans %}Owner{% endtrans %}
-        {% if pending_github_publisher_form.owner.flags.required %}
-        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
-        {% endif %}
-      </label>
-      {{ pending_github_publisher_form.owner(placeholder=gettext("owner"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", aria_describedby="owner-errors") }}
-      <p class="form-group__help-text">
-        {% trans %}The GitHub organization name or GitHub username that owns the repository{% endtrans %}
-      </p>
-      <div id="owner-errors">
-        {{ field_errors(pending_github_publisher_form.owner) }}
-      </div>
-    </div>
-    <div class="form-group">
-      <label for="repository" class="form-group__label">
-        {% trans %}Repository name{% endtrans %}
-        {% if pending_github_publisher_form.repository.flags.required %}
-        <span class="form-group__required">{% trans %}(required){% endtrans %}</span>
-        {% endif %}
-      </label>
-      {{ pending_github_publisher_form.repository(placeholder=gettext("repository"), autocomplete="off", autocapitalize="off", spellcheck="false", class_="form-group__field", **{"aria-describedby":"repository-errors"}) }}
-      <p class="form-group__help-text">
-      {% trans %}The name of the GitHub repository that contains the publishing workflow{% endtrans %}
-      </p>
-      <div id="repository-errors">
-        {{ field_errors(pending_github_publisher_form.repository) }}
-      </div>
+  <div class="horizontal-tabs" data-controller="horizontal-tabs" data-horizontal-tabs-index="0">
+    <div class="tabbar">
+      <button data-target="horizontal-tabs.tab" data-action="horizontal-tabs#change" class="tab is-active">GitHub</button>
+      <button data-target="horizontal-tabs.tab" data-action="horizontal-tabs#change" class="tab">Google</button>
```

Comment:
Deprecation warnings in Console: `The data-target attribute is deprecated and will be removed in a future version of Stimulus.`

```suggestion
      <button data-horizontal-tabs-target="tab" data-action="horizontal-tabs#change" class="tab is-active">GitHub</button>
      <button data-horizontal-tabs-target="tab" data-action="horizontal-tabs#change" class="tab">Google</button>
```